WebContent/js/components/TopBar.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { withStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import Avatar from '@material-ui/core/Avatar';
+import SettingsIcon from '@material-ui/icons/Settings';
+import Popover from '@material-ui/core/Popover';
+import Tooltip from '@material-ui/core/Tooltip';
+import SettingForm from './Setting';
+
+import JesIcon from '../../img/explorer-JES.png';
+
+const styles = {
+    customizeToolbar: {
+        minHeight: 32,
+        maxHeight: 32,
+    },
+    small: {
+        width: 24,
+        height: 24,
+    },
+};
+
+class TopBar extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            auth: true,
+            anchorEl: null,
+        };
+    }
+
+    handleClick = event => {
+        this.setState({ anchorEl: event.currentTarget });
+    };
+
+    handleClose = () => {
+        this.setState({ anchorEl: null });
+    };
+
+    render() {
+        const { anchorEl } = this.state;
+        const { validated, username, classes } = this.props;
+        const open = Boolean(anchorEl);
+        const id = open ? 'simple-popover' : undefined;
+        return (
+            <AppBar position="static" >
+                <Toolbar
+                    className={classNames(classes.customizeToolbar)}
+                    variant="dense"
+                >
+                    <Typography type="title" color="inherit" style={{ flex: 1 }}>
+                        <IconButton color="inherit">
+                            <Avatar className={classes.small} src={JesIcon} />
+                        </IconButton>
+                        JES Explorer
+                    </Typography>
+                    <Tooltip title="Setting" placement="bottom">
+                        <IconButton color="inherit" aria-describedby={id} onClick={this.handleClick}>
+                            <SettingsIcon />
```

Comment:
With the height of the app bar now smaller i think the settings and avatar icons should be a tiny bit smaller, currently there's not enough padding on the top and bottom of them in my opinion